### DISCUSSION TITLE
stat_table_card and stat_list_card as Components

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1485] [Ideas] [answered]
Proposing to make a few cards a `Component` so that they can be used with other `Component`s (within a `form_card`)

Specifically,  stat_table_card and stat_list_card

Example use cases:
- additional text other than title and subtitle
- markdown text
- button for a "closable" card of stat_list

Example:
![image](https://user-images.githubusercontent.com/64787868/172073281-74cc68da-5ae9-42a2-a82f-bb99ca1d0b81.png)

**Comment by azim-b** [comment]
@lo5 suggested this. Is this doable?

**Comment by mturoci** [comment, marked as answer]
Seems reasonable, feel free to file a feature request.

**Reply by azim-b** [reply]
thanks @mturoci
